default.nix

```diff
@@ -206,12 +206,29 @@ let
     load-config-from-site = writeTextDir "share/emacs/site-lisp/default.el" ''
       (message "doom-emacs is not placed in `doom-private-dir',
       loading from `site-lisp'")
-      (when (> emacs-major-version 26)
-            (load "${doom-emacs}/early-init.el"))
+      ${lib.optionalString (!isEmacs29) ''
+      (load "${doom-emacs}/early-init.el")
       (load "${doom-emacs}/core/core-start.el")
+      ''}
```

Comment:
Indent for the inner `''` would be nice since we are usually looking at the Nix source and not the output.